File: app/src/main/java/com/sicsha/Main/MainContract.java
package com.sicsha.Main;

import com.sicsha.Common.BaseAdapter;
import com.sicsha.Common.BasePresenter;
import com.sicsha.Common.BaseView;

/**
 * Created by kiyoungLee on 2017-10-29.
 */

public interface MainContract {

    interface ActivityView extends BaseView<Presenter> {

    }

    interface Presenter extends BasePresenter{

    }
}

File: app/src/main/java/com/sicsha/Main/MainContract.kt
package com.sicsha.Main

import android.support.v7.widget.RecyclerView
import com.sicsha.Common.BasePresenter
import com.sicsha.Main.Adapter.MainMenuTypeAdapter

/**
 * Created by kiyoungLee on 2018-01-10.
 */
interface MainContract {

    interface ActivityView{
        fun setAdapter(adapter: MainMenuTypeAdapter)
    }

    interface Presenter: BasePresenter{

    }
}